### PR TITLE
style: fix header on account/profile page

### PR DESCRIPTION
**Pull request by sugarpusher**

### CODE REVIEW

File: packages/ui/src/routes/profile/+page.svelte [hunk 1]
```diff
@@ -24,16 +24,13 @@
 			error = err as Error
 		}
 	}
+
+	export let onBack: () => unknown = () => history.back()
```
Comment (vojtechsimetka) [inline packages/ui/src/routes/profile/+page.svelte:28]:
```suggestion
```

File: packages/ui/src/routes/profile/+page.svelte [hunk 2]
```diff
@@ -24,16 +24,13 @@
...
 
 <svelte:window bind:scrollY={y} />
 
-<div class={`header ${y > 0 ? 'scrolled' : ''}`}>
-	<div>
-		<Button icon={Undo} on:click={() => history.back()} />
-		<h1>Account</h1>
-	</div>
-</div>
+<Header title="Account" {onBack} />
```
Comment (vojtechsimetka) [inline packages/ui/src/routes/profile/+page.svelte:33]:
```suggestion
<Header title="Account" onBack={() => history.back()} />
```

File: packages/ui/src/routes/profile/+page.svelte (hunk 1 above)
Comment (vojtechsimetka) [reply packages/ui/src/routes/profile/+page.svelte:28]: While it is fine to define variables and functions that you later use, it does not usually make sense if they are this small and only used once or twice.

Also (and this is svelte thing) you should not use the `export` here. The world `export` only makes sense in components and means that you are defining the variable/function as a property that can be passed to the component. `+page.svelte` is not instantiated by us so we do not pass it any properties.

**Review by vojtechsimetka — CHANGES_REQUESTED** [review]

File: packages/ui/src/routes/profile/+page.svelte (hunk 1 above)
Comment (sugarpusher) [reply packages/ui/src/routes/profile/+page.svelte:28]: Perfect! Got it.

**Pull request merged**